### ISSUE TITLE
Version 1.3.0 missing js files

### ISSUE THREAD

**Issue by ninnjak** [joshbuddy/exif-be-gone#20]
The latest version contains only ts files causing the following error:

```
Error: Cannot find module '/node_modules/exif-be-gone/index.js'. Please verify that the package.json has a valid "main" entry

code: 'MODULE_NOT_FOUND',
path: ‘/node_modules/exif-be-gone/package.json',
requestPath: 'exif-be-gone'
```

**Comment by joshbuddy** [comment]
Ah I’ll fix this in a bit. Thanks!

**Comment by joshbuddy** [comment]
Hey @ninnjak should all be good now, published as 1.3.1
